### DISCUSSION TITLE
Pull requests?

### DISCUSSION THREAD

**Discussion by david-southern** [DSchroer/dslcad#16] [General]
First off, let me say that I love this project!  It seems much more expressive than OpenSCAD, which I also love. :)

I've only been playing around with it for a day or so, but I've already seen a few things that I'd like to add as suggestions.  Given that, I thought I'd ask, are you open to pull requests for new features or bugfixes?

Don't want to be pushy about your project, or add work to you in dealing with pull requests, but I really like what I've seen so far, and would be interested in contributing.

Thanks!
David

**Comment by DSchroer** [comment]
Hey David. I'm really happy to hear that you enjoy the project. 

I would be happy to have pull requests for bugs or new features. Feel free to submit them. That said, depending on the size of the change I wouldn't want you to commit too much time before it sees the light of day. So for potentially controversial changes please open a discussion or issue about them first so that we are all able to participate. 

Looking forward to your contributions and growing this project. 

Regards,
Dominick
